crates/meroctl/src/cli/context/identity.rs

```diff
@@ -18,6 +19,30 @@ use crate::output::ErrorLine;
 mod alias;
 mod generate;
 
+pub mod grant;
+pub mod revoke;
```

Comment:
```suggestion
mod grant;
mod revoke;
```

erroneous separation & maybe I'm missing something, but I don't think these need to be `pub`?